README.md

```diff
@@ -18,8 +18,8 @@ Recommended before install:
 You will need to have the following installed on your machine before following the commands below:
 
 1. Ruby v3.2.5 (see `.ruby-version` or `.tool-versions`), [Installation guides](https://www.ruby-lang.org/en/documentation/installation/)
-1. Node v20.9.0 (see `.nvmrc` or `.tool-versions`), [Installation guides](https://nodejs.org/en/download/)
-1. Bundler v2.2.0 (see `.bundler-version`) [Installation guides](https://bundler.io/guides/using_bundler_in_applications.html#getting-started---installing-bundler-and-bundle-init)
+1. Node LTS (see `.nvmrc` or `.tool-versions`), [Installation guides](https://nodejs.org/en/download)
```

Comment:
**question:** The node version number was outdated here. I replaced the specific version number with `LTS` because that is what we have listed in [.nvmrc](https://github.com/uswds/uswds-site/blob/main/.nvmrc). However, I noticed that [.tool-versions](https://github.com/uswds/uswds-site/blob/main/.tool-versions#L2) lists 20.11.0, which is also LTS. Do we have a preference for which one to list in the README?

Reply:
Why not have it match the change in uswds/uswds#6211?

Reply:
Or is something like this better? We could do the same for the listed Bundler version.
```suggestion
1. Node (see [.nvmrc](https://github.com/uswds/uswds-site/blob/main/.nvmrc) or [.tool-versions](https://github.com/uswds/uswds-site/blob/main/.tool-versions) for version number), [Installation guides](https://nodejs.org/en/download)
```

Reply:
That looks good. Should we point to `main` instead of `develop`?

```suggestion
1. Node (see [.nvmrc](https://github.com/uswds/uswds-site/blob/main/.nvmrc) or [.tool-versions](https://github.com/uswds/uswds-site/blob/main/.tool-versions) for version number), [Installation guides](https://nodejs.org/en/download)
```